src/components/Header/Header.js

```diff
@@ -6,6 +6,8 @@ import { translate } from 'react-i18next'
 import { getUser } from '../../fetch/fetch'
 import { waitForDataAndSetState, cancelAllPromises } from '../../helpers/components'
 
+import logoMarianne from './img/logo_marianne.svg'
```

Comment:
Rename the folder to `images` and don’t use underscores for filenames (prefer dashes (`-`)), also this can be just `./images/marianne.svg`.